payments-model/src/main/java/com/stripe/android/model/ConsumerPaymentDetails.kt

```diff
@@ -62,27 +62,6 @@ data class ConsumerPaymentDetails(
 
         companion object {
             const val TYPE = "card"
-
-            /**
-             * Reads the address from the [PaymentMethodCreateParams] mapping format to the format
-             * used by [ConsumerPaymentDetails].
-             */
-            internal fun getAddressFromMap(
-                cardPaymentMethodCreateParamsMap: Map<String, Any>
-            ): Pair<String, Any>? {
-                val billingDetails =
-                    cardPaymentMethodCreateParamsMap["billing_details"] as? Map<*, *>
-                val address = billingDetails?.get("address") as? Map<*, *>
-                return address?.let {
-                    // card["billing_details"]["address"] becomes card["billing_address"]
-                    "billing_address" to mapOf(
-                        // card["billing_details"]["address"]["country"]
-                        // becomes card["billing_address"]["country_code"]
-                        "country_code" to it["country"],
-                        "postal_code" to it["postal_code"]
-                    )
-                }
-            }
```

Comment:
This was duplicated. Now we rely on `getConsumerPaymentDetailsAddressFromPaymentMethodCreateParams`.